cases/views.py

```diff
@@ -71,6 +72,22 @@ def case_list_staff(request):
 
     Case.objects.prefetch_timeline(qs)
 
+    if request.GET.get("updates"):
+        updates = 0
+        time = int(request.GET.get("updates"))
+        for case in qs:
+            if case.modified.timestamp() > time:
+                updates += 1
+        if updates == 20:
+            updates = "20+ updates"
+        elif updates == 1:
```

Comment:
I will change this, good idea, but to `updates = ngettext(f"{updates} update", f"{updates} updates", updates)` because no-one should use pluralize ever ;-)